liqd_product/apps/users/templates/liqd_product_users/user_detail.html

```diff
@@ -34,6 +34,7 @@ <h1 class="u-first-heading">{{ object.username }}</h1>
                 {% endif %}
             </ul>
 
+            {% comment %}
```

Comment:
could you pleas add a short description why this is commented and not removed to the code